src/main/scala/org/constellation/consensus/Round.scala

```diff
@@ -295,13 +322,22 @@ class Round(roundData: RoundData, arbitraryTransactions: Seq[(Transaction, Int)]
 
       val checkpointBlock = sameBlocks.head._2
 
+      dao.metrics.incrementMetric("acceptMajorityCheckpointBlockSelectedCount_" + selectedCheckpointBlocks.size)
+
       val finalFacilitators = selectedCheckpointBlocks.keySet.map(_.id).toSet
-      val cache = CheckpointCache(Some(checkpointBlock),
-        height = checkpointBlock.calculateHeight())
-      broadcastSignedBlockToNonFacilitators(FinishedCheckpoint(cache, finalFacilitators))
+      val cache = CheckpointCache(Some(checkpointBlock), height = checkpointBlock.calculateHeight())
+      log.debug(s"[${dao.id.short}] accepting majority checkpoint block ${checkpointBlock.baseHash}  " +
+        s" with txs ${checkpointBlock.transactions.map(_.hash)} " +
```

Comment:
`baseHash} _ _ " + s" _ with `
Not relevant but this looks like it's an artifact of a previous edit.

Reply:
nope, it's fine. https://docs.scala-lang.org/overviews/core/string-interpolation.html

Reply:
or you mean - spaces? it's `log.debug` so it's not so important